src/Legacy/Drupal7.php

```diff
@@ -11,7 +11,213 @@
  * Defines the Drupal 7 legacy service.
  */
 class Drupal7 {
+
+
+  /**
+   * @method void flood_register_event
+   * @method void flood_clear_event
+   * @method bool flood_is_allowed
```

Comment:
Also lets add all other methods we use here in this PR, so we have 0 new scrutinizer CI issues. :)